appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -668,10 +668,13 @@ Blockly.Msg.en.switch_language_to_english = {
 
     Blockly.Msg.LANG_TEXT_CHANGECASE_OPERATOR_UPPERCASE = 'upcase';
     Blockly.Msg.LANG_TEXT_CHANGECASE_OPERATOR_DOWNCASE = 'downcase';
+    Blockly.Msg.LANG_TEXT_CHANGECASE_OPERATOR_TITLECASE = 'titlecase';
     Blockly.Msg.LANG_TEXT_CHANGECASE_TOOLTIP_UPPERCASE = 'Returns a copy of its text string argument converted to uppercase.';
     Blockly.Msg.LANG_TEXT_CHANGECASE_HELPURL_UPPERCASE = '/reference/blocks/text.html#upcase';
-    Blockly.Msg.LANG_TEXT_CHANGECASE_TOOLTIP_DOWNCASE = 'Returns a copy of its text string argument converted to lowercase.';
+    Blockly.Msg.LANG_TEXT_CHANGECASE_TOOLTIP_DOWNCASE = 'Returns a copy of its text string argument consverted to lowercase.';
```

Comment:
Unintended change?

Reply:
oh apologies. It was at that time when I was not able to understand how the server works so and, was trying to get the changes reflected on the website. will remove this asap.